Editors/vscode/package.json

```diff
@@ -46,6 +46,11 @@
                     "default": "sourcekit-lsp",
                     "description": "The path of the sourcekit-lsp executable"
                 },
+                "sourcekit-lsp.serverOptions": {
+                    "type": "array",
```

Comment:
We could add a schema for the array elements as well:
```
"items": {
    "type": "string"
},
```

Reply:
Done, but just an FYI, that changes the settings UI so that now there's an 'Add Item' button so adding both an option and a value requires entering multiple "Items". Not a big deal, but hopefully that's obvious for others.

![Screen Shot 2020-04-21 at 12 12 45 PM](https://user-images.githubusercontent.com/11216295/79899052-6f77aa80-83c9-11ea-95d8-0b37e810b10f.png)